packages/connect-examples/new-example/app/components/ui/DeviceActionAnimation.tsx

```
@@ -0,0 +1,179 @@
+import React from "react";
```

ESLint

[error] Resolve error: typescript with invalid interface loaded as resolver

(import/namespace)